Formula/libpq.rb

```diff
@@ -0,0 +1,24 @@
+class Libpq < Formula
+  desc "Postgres C API library"
+  homepage "https://www.postgresql.org/docs/9.6/static/libpq.html"
+  url "https://ftp.postgresql.org/pub/source/v9.6.3/postgresql-9.6.3.tar.bz2"
+  sha256 "1645b3736901f6d854e695a937389e68ff2066ce0cde9d73919d6ab7c995b9c6"
+
+  keg_only "conflicts with postgres formula"
+
+  depends_on "openssl"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--prefix=#{libexec}/libpq",
+                          "--with-openssl"
+    system "make"
+    system "make", "-C", "src/bin", "install"
+    system "make", "-C", "src/include", "install"
+    system "make", "-C", "src/interfaces", "install"
```

Comment:
Any chance we can run `make install` instead of the three different invocations? Or maybe even combine with `make` above?